.github/workflows/ci.yaml

```diff
@@ -86,46 +83,36 @@ jobs:
           restore-keys: |
             gobuild-linux-amd64
 
-      - name: Checkout charts repository for deployments
-        uses: actions/checkout@v2
-        with:
-          repository: newrelic/helm-charts
-          path: helm-charts-newrelic
-
-      - name: Install kind
-        run: go install sigs.k8s.io/kind@v0.11.1
-
       - name: Create k8s Kind Cluster
-        run: make -C newrelic-k8s-metrics-adapter kind-up
+        run: make kind-up
 
       - name: Run integration tests
         env:
           NEWRELIC_API_KEY: "${{ secrets.NEWRELIC_API_KEY }}"
           NEWRELIC_ACCOUNT_ID: "${{ secrets.NEWRELIC_ACCOUNT_ID }}"
-        run: make -C newrelic-k8s-metrics-adapter test-integration
+        run: make test-integration
 
       - name: Build Docker image
-        run: make -C newrelic-k8s-metrics-adapter image-push IMAGE_REPO=localhost:5000/newrelic-k8s-metrics-adapter
+        run: make image-push IMAGE_REPO=localhost:5000/newrelic-k8s-metrics-adapter
 
       - name: Create Helm release
         env:
-          KUBECONFIG: ./newrelic-k8s-metrics-adapter/kubeconfig
+          KUBECONFIG: ./kubeconfig  # Generated by `make kind-up`
           NEWRELIC_API_KEY: "${{ secrets.NEWRELIC_API_KEY }}"
           NEWRELIC_ACCOUNT_ID: "${{ secrets.NEWRELIC_ACCOUNT_ID }}"
         run: |
-          make -C newrelic-k8s-metrics-adapter generate-local-values
-          helm install newrelic-k8s-metrics-adapter ./helm-charts-newrelic/charts/newrelic-k8s-metrics-adapter \
-          --values ./newrelic-k8s-metrics-adapter/values-dev.yaml \
-          --values ./newrelic-k8s-metrics-adapter/values-local.yaml \
+          make generate-local-values
+          helm install newrelic-k8s-metrics-adapter ./charts/newrelic-k8s-metrics-adapter \
           --set image.repository=localhost:5000/newrelic-k8s-metrics-adapter \
           --set image.pullPolicy=IfNotPresent \
-          --wait
+          --values values-dev.yaml --values values-local.yaml \
```

Comment:
I know that it was already like that, but I find it a bit confusing to have values-dev.yaml, values-local.yaml, values.yaml and ci/values.yaml

Reply:
You are right but I would like to don't go out of scope of the PR which is about changing the CI.

Changing this means changing also the behavior of `Tiltfile` and `Makefile` so `kind`, the local registry, values and all the development tooling is more readable.

I would prefer other PR for that.

Reply:
it totally makes sense